### ISSUE TITLE
react-perf/jsx-no-jsx-as-prop listed as part of `recommended` in README.md, but isn't

### ISSUE THREAD

**Issue by bryan-codaio** [cvazac/eslint-plugin-react-perf#55]
Commit 2d818141d82 added the new `jsx-no-jsx-as-prop ` rule and added it immediately to the `recommended` configuration. However, commit 8f121dac1e0 then removed it from recommended, notably without updating the README.md.

I could submit a pull request to update the README.md to match the code. Before I do that, though, I was wondering if there's any context on why this was not a recommended rule? Commit 8f121dac1e0 doesn't provide any context

**Comment by cvazac** [comment]
thanks for your patience! 

I don't know why `jsx-no-jsx-as-prop` was removed from recommended. I think it should be added back, wdyt?

**Comment by cvazac** [comment]
@bryan-codaio thanks for your patience!

I don't know why jsx-no-jsx-as-prop was removed from recommended. I think it should be added back, wdyt?

**Comment by bryan-codaio** [comment]
yeah, it strikes me that it ought to be in the recommended config unless there's a specific reason otherwise
